### PR TITLE
Add profile pages

### LINKED ISSUE #21: Profile page
As a mentee, mentor, or admin, I would like to see my profile as well as other people's profiles. On their profile, I want their Google login picture, their name, their bio, and their contact information.

To implement this, we should create a page that contains the user's id as a route parameter `/users/:id` or some other naming method.

A user should be able to see every other user right now. This will require a single call to a function in a user router that gets the profile based on id.

### PR DESCRIPTION
**Pull request by nad2040**
closes #21

Adds profile page under /profiles/[id] route.

### CODE REVIEW

File: src/pages/profiles/[id].tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import { api } from "../../utils/api";
+import { useRouter } from "next/router";
+import Head from "next/head";
+import { Button, ButtonGroup, Box } from "@mui/material";
+import { Profile } from "@prisma/client";
+
+type ProfileProp = {
```
Comment (ulrokx) [inline src/pages/profiles/[id].tsx:7]: ButtonListProps

File: src/pages/profiles/[id].tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  profile: Profile;
+};
+
+const ButtonsList = (prop: ProfileProp) => {
```
Comment (ulrokx) [inline src/pages/profiles/[id].tsx:11]: you can destructure { profile }

**Review by ulrokx — CHANGES_REQUESTED** [review]
Looks good

File: src/pages/profiles/[id].tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      <Head>
+        <title>Asian American Dream Admin</title>
+      </Head>
+      <main className="flex min-h-screen flex-col items-center bg-gradient-to-b from-midnight-sky to-aero">
+        <h1 className="mt-12 text-5xl font-bold text-[#F9F8F5]">
+          Asian American Dream Admin
+        </h1>
+        <div className="mx-14 mt-20 flex flex-grow flex-col items-center justify-between">
+          <Box
+            sx={{
```
[the remark this answers is not in the document]
Comment (ulrokx) [reply src/pages/profiles/[id].tsx:55]: For clarification, tailwind styles means using classNames and tailwind classes.

**Comment by ShawnAviles** [comment]
Need this to work on the Profile Editing screen @ulrokx. I can address the styling comments while working on #126.

**Review by ulrokx — APPROVED** [review]

**Pull request merged**